### PR TITLE
add Julia support

### PR DESCRIPTION
**Pull request by mbesancon**
Just as #31 this is a first draft to add Julia support. The core fx package was not yet modified.
An example function was added in example, using a typed interface with an Input struct.
`images/julia` contains an app.jl using a minimalist HTTP server, the function file and the docker image.

### CODE REVIEW

**Comment by metrue** [comment]
I will test it out soon, I will merge it soon if everything works as expected.

Thank you so much, this is a great PR.

File: images/julia/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM julia:latest
+
+COPY . /app
+
+RUN apt-get update && apt-get install -y gcc apt-utils unzip make libhttp-parser-dev
+RUN julia /app/deps.jl
+
+CMD julia /app/app.jl
+EXPOSE 8080
```
Comment (metrue) [inline images/julia/Dockerfile]: @mbesancon can you expose 3000 instead 8080, let's keep same port like the other Dockerfile of fx/images

File: images/julia/app.jl [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    if length(req.data) == 0
+        res.status = 400
+    else
+        parsed_data = JSON.Parser.parse(join([Char(v) for v in req.data]))
+        data = Unmarshal.unmarshal(Input, parsed_data)
+        string(fx(data))
+    end
+end
+
+run(srv, 8080)
```
Comment (metrue) [inline images/julia/app.jl]: prefer 3000.

File: images/julia/Dockerfile (hunk 1 above)
Comment (mbesancon) [reply images/julia/Dockerfile]: Yep, I'll change that tomorrow

**Comment by mbesancon** [comment]
@metrue I changed the port in the Docker and app.jl to 3000

**Pull request merged**